15314f1ddb882700dc9fabc5ca961943/update/ecc_agent_script_include_63973142db8c6700dc9fabc5ca961947.xml

```diff
@@ -55,11 +55,19 @@ IllumioManageAsyncJobs.prototype = {
 
         this.pceHttpClient = new IllumioHTTPClient(this.pceUrl, this.pceUsername, this.pcePassword, this.protocol, this.pceMIDProxy, this.retryParams);
         this.snHttpClient = new IllumioHTTPClient(this.snowUrl, this.snowUsername, this.snowPassword, "443", null, this.retryParams);
+		this.illumio_jobId = probe.getParameter("glide.jms.illumio_scheduled_job_id");
     },
 
     run: function() {
+		
+	if (
+            this.utils.checkCancelledJobStatus(this.illumio_jobId, this.snowUrl, this.snowUsername, this.snowPassword)
```

Comment:
this.utils.checkCancelledJobStatus function returns the boolean value, if job is cancelled it will return true otherwise false.
If job is cancelled then we simply have to stop the further processing and that's why we have added return statement.